https://git.savannah.gnu.org/cgit/rcs.git/commit/?h=p&id=8883c4f5a29be18e9ea09bd27a7b660830de45bb
https://bugs.gentoo.org/820230

From: Thien-Thi Nguyen <ttn@gnu.org>
Date: Fri, 23 Oct 2020 09:23:49 -0400
Subject: [v] Don't test signal handling if stdin not ok.

<https://mail.gnu.org/archive/html/bug-rcs/2020-10/msg00014.html>

* tests/t632: If stdin is not open and connected to
a tty, skip the signal handling portion of the test.
--- a/tests/t632
+++ b/tests/t632
@@ -40,6 +40,15 @@ echo | co -l -I $w \
 # (This is skipped if GNU coreutils timeout(1) is not available.)
 ##
 
+# <https://mail.gnu.org/archive/html/bug-rcs/2020-10/msg00014.html>
+# The timeout test needs co(1) to block on input.
+# If stdin is not open and connected to a tty, skip out.
+if test -t 0 ; then
+    echo STDIN OK
+else
+    exit 0
+fi
+
 # TODO: Don't be lame!  Pick one:
 # (a) Mimic timeout(1) w/ sh commands.
 # (b) Incorporate heart of timeout(1) into ./btdt and use that.
cgit v1.2.1
